jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/vsd/VsdReceiver.java

```diff
@@ -0,0 +1,100 @@
+package us.dot.its.jpo.ode.vsd;
+
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.util.Arrays;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.oss.asn1.AbstractData;
+
+import us.dot.its.jpo.ode.OdeProperties;
+import us.dot.its.jpo.ode.bsm.BsmReceiver2;
+import us.dot.its.jpo.ode.j2735.dsrc.BasicSafetyMessage;
+import us.dot.its.jpo.ode.j2735.semi.ServiceRequest;
+import us.dot.its.jpo.ode.j2735.semi.VehSitDataMessage;
+import us.dot.its.jpo.ode.plugin.j2735.oss.OssBsmPart2Content.OssBsmPart2Exception;
+
+public class VsdReceiver extends BsmReceiver2 {
```

Comment:
In order to reuse the publish BSM method. This is essentially an extension of BsmReceiver.